manager/client/main.go

```diff
@@ -22,7 +23,7 @@ func run(timeout time.Duration, cmd string, args ...string) {
 	job := exec.CommandContext(ctx, cmd, args...)
 	err := job.Run()
 	if ctx.Err() == context.DeadlineExceeded {
-		log.Fatalf("Job failed to complete after %d seconds: %s %s \n", timeout, cmd, strings.Join(args, " "))
+		log.Fatalf("Job failed to complete after %s: %s %s \n", timeout, cmd, strings.Join(args, " "))
```

Comment:
nit: if you use `%v`, you can just leave `args` without the strings.Join()

Reply:
It still prints with brackets 
```
vivado [-mode batch -source fpga/projects/lupa/ip/generate_ip_bazel.tcl -tclargs xcvu9p-fsgd2104-2L-e lupa bazel-out/k8-fastbuild/bin/fpga/projects/lupa/ip/vcu1525/ip_axis_switch_S8xM1 fpga/projects/lupa/ip fpga/projects/lupa/ip/common/ip_axis_switch_S8xM1.tcl -notrace]
```
which I do not want.